content/doc/installation/run-as-service-in-ubuntu.en-us.md

```diff
@@ -0,0 +1,113 @@
+---
+date: "2017-07-21T12:00:00+02:00"
+title: "Run as service in Linux"
+slug: "linux-service"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Linux service"
+    weight: 20
+    identifier: "linux-service"
+---
+
+### Run as service in Ubuntu 16.04 LTS  
+ 
+#### Using systemd  
+
+Run below command in terminal:  
+```
+sudo vim /etc/systemd/system/gitea.service
+```  
+
+Add the following code to the file, [https://github.com/go-gitea/gitea/blob/master/contrib/systemd/gitea.service](https://github.com/go-gitea/gitea/blob/master/contrib/systemd/gitea.service):  
+```
```

Comment:
Please don't include the full copy here, as it will become outdated at some point